mapiproxy/services/plugins/dovecot/ChangeLog

```diff
@@ -0,0 +1 @@
+ * Removed openchange_cn option, always using email
```

Comment:
As we are not using mail attribute, but proxyAddresses one, we must specify that in the changelog to whomever read this...